src/simulators/matrix_product_state/matrix_product_state_internal.cpp

```diff
@@ -1327,7 +1413,7 @@ reg_t MPS::sample_measure_using_probabilities_internal(const rvector_t &rnds,
   #pragma omp parallel if (SHOTS > omp_threshold_ && omp_threads_ > 1) num_threads(omp_threads_)
     {
       #pragma omp for
-  for (int_t i = 0; i < SHOTS; ++i) {
+      for (int_t i = 0; i < static_cast<int_t>(SHOTS); ++i) {
```

Comment:
Here in GitHub the indentation looks wrong

Reply:
fixed the indentation